analyses/molecular-subtyping-LGAT/00-subset-files-for-LGAT.R

```diff
@@ -0,0 +1,44 @@
+# K S Gaonkar 2020
+# Subsets the consensus mutation file to biospecimens where short_histology == LGAT
+
+# Look for git root folder
+root_dir <- rprojroot::find_root(rprojroot::has_dir(".git"))
+
+# get subset folder
+subset_dir<-file.path(root_dir,"analyses","molecular-subtyping-LGAT","lgat-subset")
```

Comment:
Let's make sure this directory gets created if it doesn't exist with:
```r
if (!dir.exists(subset_dir)) {
  dir.create(subset_dir)
}
```

You're also able to do
```R
 dir.create(subset_dir, showWarnings = FALSE)
```
but I like how spelled out the other one is. A matter of personal preference.